client/src/pages/course/schedule.tsx

```diff
@@ -46,6 +54,11 @@ export function SchedulePage(props: CoursePageProps) {
 
   const viewMode = scheduleViewMode as ViewMode;
   const ScheduleView = mapScheduleViewToComponent[viewMode] || TableView;
+  const filteredData = isHideOldEvents ? eventsWithoutOld : data;
+
+  const hideOldEvents = () => {
```

Comment:
this function doesn't really hide events, it toggles a flag, so let's rename it to avoid any confusing: hideOldEvents  -> toggleHideOldEvents

Reply:
fix